### DISCUSSION TITLE
Issues with Converting Atlantis Template from PrimeFlex to Tailwind – Broken Design Elements

### DISCUSSION THREAD

**Discussion by RadwaMosaad** [primefaces/community#2572] [PrimeVue Templates] [answered]
Hello everyone,

I recently purchased the Atlantis template from PrimeVue and expected it to be compatible with Tailwind CSS, similar to the Sakai template I tested earlier. After discovering that Atlantis actually uses PrimeFlex, I followed the official documentation provided by PrimeTek to convert the template from PrimeFlex to Tailwind. The guide I used can be found here: (https://primevue.org/guides/primeflex/)

Problem
After completing the conversion, I encountered numerous design issues. Specifically, many elements on the screen are now broken, with layouts not rendering correctly, overlapping components, and general design crashes. I followed the documentation closely, but the result is far from what I expected.

Steps Taken
Downloaded the Atlantis template from PrimeVue.
Followed the guide to replace PrimeFlex utilities with Tailwind equivalents, as outlined in the documentation (https://primevue.org/guides/primeflex/)
Reviewed the Tailwind setup to ensure it was configured correctly within template project.
Rechecked my changes to ensure I hadn't missed any steps or dependencies mentioned in the documentation.

Observations
The grid system is particularly affected, with elements either collapsing or overlapping.
Margins, paddings, and spacings seem inconsistent and do not align with the original design.
Some PrimeVue components are not rendering as expected after the conversion.

Request for Assistance
I would appreciate any insights, suggestions, or examples from anyone who has successfully completed a similar conversion. 
Specifically:
Has anyone faced similar issues when converting from PrimeFlex to Tailwind?
Are there any common pitfalls or additional steps that might not be covered in the official documentation?

Thank you in advance for your help!
![Uploading Design Failure-1.jpg…]()

**Comment by cantin-f** [comment]
You should wait, Atlantis (Tailwind version) is planned for September. That's the last estimation I've seen.

**Comment by tugcekucukoglu** [comment, marked as answer]
The new version of Atlantis with Tailwind can be downloaded from the PrimeStore.
